contracts/strategies/qv-simple/QVSimpleStrategy.sol

```diff
@@ -40,8 +76,9 @@ contract QVSimpleStrategy is QVBaseStrategy {
     /// ===============================
 
     /// @notice Initialize the strategy
-    /// @param _poolId The pool id
-    /// @param _data The data
+    /// @dev Only `Allo` can call this function
+    /// @param _poolId The ID of the pool
+    /// @param _data The initialization data for the strategy
```

Comment:
If possible, can we add a custom with the data details here? Ex from RFP: /// @custom:data (uint256 voteThreshold, InitializeParams params)